install_sct

```diff
@@ -687,9 +687,12 @@ if [[ -n "$NO_SCT_BIN_INSTALL" ]]; then
   print warning "WARNING: SCT binaries will not be (re)-installed"
 else
   print info "Installing binaries..."
-  run sct_download_data -d "binaries_${OS}" -k
+  # shellcheck disable=SC2015  # it would be very strange if print (echo) failed and triggered the || case
+  run sct_download_data -d "binaries_${OS}" -k \
+    && print info "All requirements installed!" \
+    || print error "Failed to install 'binaries_${OS}'. Please re-run 'sct_download_data -d binaries_${OS}' after installation."
```

Comment:
> If this is just a line buried in the middle of the install log, with no bold red text at the end of the install process, I think people will ignore it.

I think `print error` is what adds the bold red text? But, I on reflection I agree that this would get buried.